src/framework/entity.js

```diff
@@ -601,6 +604,24 @@ class Entity extends GraphNode {
         return clone;
     }
 
+    _getSortedComponents() {
+        _sortedArray.length = 0;
+
+        let needSort = false;
+        for (const component of Object.values(this.c)) {
+            if (component.constructor.order !== 0) {
+                needSort = true;
+            }
+            _sortedArray.push(component);
+        }
+
+        if (needSort) {
```

Comment:
Could be a tiny speed boost

```suggestion
        if (needSort && _sortedArray.length > 1) {
```